source/includes/example-clone-collection.rst

```diff
@@ -0,0 +1,9 @@
+.. code-block:: javascript
+
+   { cloneCollection: "users.profiles",
```

Comment:
We should make this a functioning example by surrounding it wiht

db.runCommand( { ... } )